packages/builder/src/components/common/DatePicker.svelte

```diff
@@ -1,32 +1,38 @@
 <script>
```

Comment:
NAB - shoud we consider putting this in BBUI ?

Reply:
Good shout - probably worth it, considering we will be using pretty much exactly the same component in standard-components